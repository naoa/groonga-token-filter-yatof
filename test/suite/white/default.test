register token_filters/yatof

table_create white_terms TABLE_HASH_KEY ShortText --normalizer NormalizerAuto
load --table white_terms
[
{"_key": "senna"}
]

tokenize TokenBigram "Hello Senna" \
  --normalizer NormalizerAuto \
  --token_filters TokenFilterWhite

