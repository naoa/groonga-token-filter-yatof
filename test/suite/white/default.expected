register token_filters/yatof
[[0,0.0,0.0],true]
table_create white_terms TABLE_HASH_KEY ShortText --normalizer NormalizerAuto
[[0,0.0,0.0],true]
load --table white_terms
[
{"_key": "senna"}
]
[[0,0.0,0.0],1]
tokenize TokenBigram "Hello Senna"   --normalizer NormalizerAuto   --token_filters TokenFilterWhite
[[0,0.0,0.0],[{"value":"senna","position":1,"force_prefix":false}]]
